gui/widgets/richtext.cpp

```diff
@@ -34,6 +34,14 @@
 
 namespace GUI {
 
+Graphics::TTFMap ttfFamily[] = {
+	{"NotoSans-Regular.ttf", Graphics::kMacFontRegular},
+	{"NotoSans-Bold.ttf", Graphics::kMacFontBold},
+	{"NotoSerif-Italic.ttf", Graphics::kMacFontItalic},
+	{"NotoSerif-Bold-Italic.ttf", Graphics::kMacFontBold | Graphics::kMacFontItalic},
+	{nullptr, 0},
```

Comment:
Once again, my suggestion is to make it `0`, not `nullptr`. This way, your checks become simpler.

Reply:
ok, I'll do it